ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -268,6 +276,13 @@ class FamilyReads extends React.PureComponent {
     locus: null,
   }
 
+  updateReads = (familyGuid, locus, sampleTypes, tissueType) => {
+    const rnaReferenceOptions = [NORM_GTEX_TRACK_OPTIONS, AGG_GTEX_TRACK_OPTIONS].map(options => (
```

Comment:
Why do we need a different default behavior for reference tracks here than we do in other cases?

Reply:
We need to show `the GTEx track for the same tissue`. So we set the default options for the GTEx tracks according to the tissue type. For the other cases, no default reference tracks are selected by default.